src/ecs/wizards/executeCommand.ts

```diff
@@ -0,0 +1,93 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ecsExecToolkitGuideUrl } from '../../shared/constants'
+import { createCommonButtons } from '../../shared/ui/buttons'
+import { createInputBox } from '../../shared/ui/inputPrompter'
+import { createQuickPick, DataQuickPickItem } from '../../shared/ui/pickerPrompter'
+import { localize } from '../../shared/utilities/vsCodeUtils'
+import { WIZARD_EXIT, Wizard, WIZARD_BACK } from '../../shared/wizards/wizard'
+import { EcsContainerNode } from '../explorer/ecsContainerNode'
+
+export interface CommandWizardState {
+    task: string
+    command: string
+    confirmation?: 'yes' | 'supress'
```

Comment:
misspelling (also in other parts of this PR)

```suggestion
    confirmation?: 'yes' | 'suppress'
```

Reply:
Can this just be a boolean? `yes` and `suppress` are kind of strange as the two available values.

Reply:
Making this boolean would be more confusing as there are 3 options for this prompt. The third option ('no') is just 'hidden' from the type as it closes the wizard. 

For clarity we could make 'no' not close the wizard and instead handle it as its own thing. I think that could read better albeit with a bit more logic.